homework-submission/src/components/Todos.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import TodoItem from './TodoItem';
+
+const StaticTodos = ({ description, deadline }) => {
```

Comment:
You're using the plural work StaticTodos here while it only renders one todo item, might be confusing